homeassistant/components/binary_sensor/hive.py

```diff
@@ -35,9 +35,24 @@ def __init__(self, hivesession, hivedevice):
         self.attributes = {}
         self.data_updatesource = '{}.{}'.format(self.device_type,
                                                 self.node_id)
-
+        self._unique_id = '{}-{}'.format(self.node_id, self.device_type)
         self.session.entities.append(self)
 
+    @property
+    def unique_id(self):
+        """Return unique ID of entity."""
+        return self._unique_id
+
+    @property
+    def device_info(self):
```

Comment:
No, this shouldn't have been included, as noted in earlier PR. `device_info` is only read if config entry is used.

Reply:
Sorry Martin, a misunderstanding on my behalf